src/jit/instrsarm.h

```diff
@@ -450,7 +450,7 @@ INST1(tbb,     "tbb",    0, 0, IF_T2_C9,  0xE8D0F000)
                                            //  Rn,Rm             T2_C9     111010001101nnnn 111100000000mmmm   E8D0 F000                
 INST1(tbh,     "tbh",    0, 0, IF_T2_C9,  0xE8D0F010)
                                            //  Rn,Rm,LSL #1      T2_C9     111010001101nnnn 111100000001mmmm   E8D0 F010                       
-INST1(ubfx,    "ubfx",   0, 0, IF_T2_D0,  0xF2C00000)
+INST1(ubfx,    "ubfx",   0, 0, IF_T2_D0,  0xF3C00000)
```

Comment:
Wow!
You fixed a bug which might be hard to find, since `0xF2C0 0000` is `movt` which may not incur illegal instruction fault.  Thanks.
And it seems that LegacyJIT don't make use of `ubfx` (fortunately ?).